File: stress/t/flt_sqrt.c
#include <stdio.h>
#include <math.h>
int main(void){
  double x,y;
  x=2.0; y=sqrt(x);
  printf("y= %f\n",y);
  x=1000000.0; y=sqrt(x);
  printf("y= %f\n",y);
  return 0;
}

File: stress/t/flt_sqrt.f
      PROGRAM FSQRT
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      X=2.0D0
      Y=SQRT(X)
      WRITE(*,'(A,F0.6)') 'y= ',Y
      X=1000000.0D0
      Y=SQRT(X)
      WRITE(*,'(A,F0.6)') 'y= ',Y
      END
